code/modules/awaymissions/maploader/reader.dm

```diff
@@ -220,9 +220,12 @@ var/global/dmm_suite/preloader/_preloader = new
 	var/atom/instance
 	_preloader.setup(attributes, path)
 
-	var/turf/T = locate(x,y,z)
-	if(T)
-		instance = new path (T)//first preloader pass
+	var/turf/oldTurf = locate(x,y,z)
+	if(ispath(path, /turf)) //Turfs use ChangeTurf
+		if(path != oldTurf.type)
+			instance = oldTurf.ChangeTurf(path)
```

Comment:
This is calling ChangeTurf() on every single turf in the away mission. Startup times are going to increase immensely as a result because ChangeTurf is expensive as fuck, and this probably also makes atmos flip out.